waterArduino.INO

```diff
@@ -22,28 +30,37 @@ void setup()
 
 void loop() 
 {
-  for(int i = 0; i < 10; i++)   //for loop to calculate average of ten soil readings
+  avgDirt = analogRead(sensPin);
+  Serial.print("first dirt value: ");
+  Serial.println(avgDirt);
+  for(int i = 0; i < numReads; i++)   //for loop to calculate average of ten soil readings
   {
     dirtVal = analogRead(sensPin);
-    delay(30*1000);
+//    delay(30*1000);
+    Serial.print("partial dirt value: ");
+    Serial.println(dirtVal);
     avgDirt += dirtVal;
+    Serial.print("sum value: ");
+    Serial.println(avgDirt);
   }
-  avgDirt = avgDirt / 10.0;     //gets average value of 10 sensor readings
-  Serial.print("Sensor value: "); //for debugging
-  Serial.print(avgDirt);
+  avgDirt = avgDirt / (numReads + 1.0);     //gets average value of 10 sensor readings
+  Serial.print("avgDirt before water: "); //for debugging
+  Serial.println(avgDirt);
 
   if(avgDirt <= dryVal)   //if the average soil value is dry
   {
     waterServo.write(waterPosition);  //start watering the plants
     while(dirtVal <= wetVal)  //while loop to keep watering until the soil is wet
     {
       dirtVal = analogRead(sensPin);
-      Serial.println(avgDirt);
-      delay(20);
+      Serial.print("dirtVal during water: ");
+      Serial.println(dirtVal);
+      delay(2000);
```

Comment:
made this delay a little slower so we could see it come up in the console easier.